nixos/modules/hardware/ledger.nix

```
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.hardware.ledger;
+
+  udevRules = pkgs.stdenv.mkDerivation {
+    name = "ledger-udev-rules";
```

This needs to have a version. Since it's not a release and just a commit from this repo, it can be formatted like
```suggestion
    pname = "ledger-udev-rules";
    version = "unstable-YYYY-MM-DD"
```

> If a package is not a release but a commit from a repository, then the version part of the name must be the **date of that (fetched) commit**. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23". 

Also using `pname` is preferred now.